classroom-dashboard/app/controllers/sections_controller.rb

```diff
@@ -16,6 +16,7 @@ def new
 		@section = Section.new
 	end
 
+	# Does this edit function work by default?
```

Comment:
thanks for catching this!
I believe it should have the following: 
@student =Student.find(params[:id])

Try running it with the above. If it doesn't work, we know theres another bug in our code.